app/ui/legacy/src/main/res/drawable/message_list_item_selection_background.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:innerRadius="0dp"
+    android:shape="ring"
+    android:thicknessRatio="2.0"
+    android:useLevel="false">
+
+    <solid android:color="?attr/colorPrimary" />
+</shape>
```

Comment:
This feels like a complicated way to create a circle.

```suggestion
<shape xmlns:android="http://schemas.android.com/apk/res/android"
    android:shape="oval">

    <solid android:color="?attr/colorPrimary"/>
</shape>
```

Reply:
I planed to use this also at another place with an outer circle. But I think it's okay to change this as I made up my mind and am going to replace the other component completely.